proto/sample_v2_round_trip.json

```diff
@@ -3262,7 +3262,7 @@
         "throttlingMethod": "devtools"
     }, 
     "environment": {
-        "benchmarkIndex": 1000, 
+        "benchmarkIndex": 1000.0, 
```

Comment:
since this is JSON, the trailing `.0` ends up in sort of in undefined behavior (it won't survive depending on the language/tool used to parse/stringify the file). Since it's a fake number anyways, should we change the `benchmarkIndex` value in `sample_v2.json` so it's always a float (e.g. 1000.5)?

Or we can leave it and worry about it when it creates a problem (again) :)